CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.8.3 (UNRELEASED)
+--------------------------
+
+- Changes installation to require kombu version 4.6 or higher.
```

Comment:
```suggestion
- Changes installation to require ``kombu`` version 4.6 or higher.
```

`kombu` version 4.6 or higher was partially required _in a way_. Perhaps it's more helpful to mention that the upper pin was removed.